Makefile

```diff
@@ -55,7 +55,19 @@ ifneq ($(BUILD_IN_CONTAINER),1)
 	CGO_ENABLED=0 GO111MODULE=on GOFLAGS='$(GOFLAGS)' GOOS=linux go build -installsuffix cgo -ldflags "-w -X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o nginx-ingress github.com/nginxinc/kubernetes-ingress/cmd/nginx-ingress
 endif
 
-container: test verify-codegen verify-crds binary certificate-and-key
+install-plus:
+ifneq (,$(findstring Plus,$(DOCKERFILE)))
+	mkdir -p tempdir && base64 nginx-repo.crt > tempdir/nginx-repo.crt && base64 nginx-repo.key > tempdir/nginx-repo.key
+DOCKER_BUILD_OPTIONS += --secret id=nginx-repo.crt,src=tempdir/nginx-repo.crt --secret id=nginx-repo.key,src=tempdir/nginx-repo.key
+endif
+
+install-plus-openshift: install-plus
+ifneq (,$(findstring PlusForOpenShift,$(DOCKERFILE)))
+	base64 rhel_license > tempdir/rhel_license
+DOCKER_BUILD_OPTIONS += --secret id=rhel_license,src=tempdir/rhel_license
+endif
+
+container: test verify-codegen verify-crds binary certificate-and-key install-plus-openshift
```

Comment:
i would choose a slightly different names for the new targets. Something like `prepare-plus-license-secret` and `prepare-rhel-license-secret` or a combined `prepare-license-secrets`.  Imho `install` reminds of `make install`, which typically installs compiled software on the machine.

Reply:
Thanks, great point, naming of things is not one of my strengths :) I've updated it now!